### PR TITLE
[Merged by Bors] - chore: redistribute tags for `fun_prop` regarding continuity across the library

### PR DESCRIPTION
**Pull request by j-loreaux**
---
I have tried to keep the commit history very clean so that reviewers can review commit-by-commit to ensure that nothing is lost.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by ericrbg** [comment]
btw, `import Mathlib` does not fix the issue; I'll see if I can spot what was missed.

**Comment by j-loreaux** [comment]
@ericrbg I found it: I placed `fun_prop` on`ContinuousWithinAt.log` instead of `ContinuousOn.log`.

Note: I have already talked with Tomas and we *will* want to put the attribute on `ContinuousWithinAt` too, it's just been an oversight so far. I didn't do it for all the `ContinuousWithinAt` lemmas this time because they weren't already marked, and I wanted to keep review of this PR reasonable.

**Comment by lecopivo** [comment]
I had a quick look through the changes and it all looks good to me! Thanks for doing this!

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

File: Mathlib/Topology/ContinuousOn.lean [hunk 1]
```diff
@@ -937,6 +943,11 @@ theorem ContinuousOn.comp {g : β → γ} {f : α → β} {s : Set α} {t : Set
   ContinuousWithinAt.comp (hg _ (h hx)) (hf x hx) h
 #align continuous_on.comp ContinuousOn.comp
 
+@[fun_prop]
+theorem ContinuousOn.comp'' {g : β → γ} {f : α → β} {s : Set α} {t : Set β} (hg : ContinuousOn g t)
```
Comment (adomani) [inline Mathlib/Topology/ContinuousOn.lean:947]: I wrote a small (buggy) script to check that nothing has been lost.  According to the script, this theorem may not have a `fun_prop` before the PR.

File: Mathlib/Topology/Defs/Filter.lean [hunk 2]
```diff
@@ -152,18 +152,21 @@ def nhdsSet (s : Set X) : Filter X :=
...
 if `f x` tends to `f x₀` when `x` tends to `x₀`. -/
+@[fun_prop]
 def ContinuousAt (f : X → Y) (x : X) :=
   Tendsto f (𝓝 x) (𝓝 (f x))
 #align continuous_at ContinuousAt
 
 /-- A function between topological spaces is continuous at a point `x₀` within a subset `s`
 if `f x` tends to `f x₀` when `x` tends to `x₀` while staying within `s`. -/
+@[fun_prop]
 def ContinuousWithinAt (f : X → Y) (s : Set X) (x : X) : Prop :=
```
Comment (adomani) [inline Mathlib/Topology/Defs/Filter.lean:163]: I wrote a small (buggy) script to check that nothing has been lost.  According to the script, this definition may not have a `fun_prop` before the PR.

File: Mathlib/Tactic/FunProp/Continuous.lean [hunk 3]
```diff
@@ -1,226 +0,0 @@
...
-  Continuous.exp
-  Continuous.cexp
-  Continuous.log
-  Continuous.pow
-  Continuous.sqrt
-
-  ContinuousAt.exp
-  ContinuousAt.cexp
-  ContinuousAt.log
-  ContinuousAt.pow
```
Comment (adomani) [inline Mathlib/Tactic/FunProp/Continuous.lean:215]: Is this the same lemma that is also tagged earlier?

File: Mathlib/Tactic/FunProp/Continuous.lean [hunk 4]
```diff
@@ -1,226 +0,0 @@
...
-  ContinuousAt.sqrt
-
-  ContinuousOn.exp
-  ContinuousOn.cexp
-  ContinuousOn.log
-  ContinuousOn.pow
```
Comment (adomani) [inline Mathlib/Tactic/FunProp/Continuous.lean:221]: Is this the same lemma that is also tagged earlier?

File: Mathlib/Tactic/FunProp/Continuous.lean [hunk 5]
```diff
@@ -1,226 +0,0 @@
...
-  ContinuousOn.nnnorm
-  ContinuousOn.norm'
-  ContinuousOn.nnnorm'
-
--- special function
-attribute [fun_prop]
-  Continuous.exp
-  Continuous.cexp
-  Continuous.log
-  Continuous.pow
```
Comment (adomani) [inline Mathlib/Tactic/FunProp/Continuous.lean:209]: Is this the same lemma that is also tagged earlier?

**Review by adomani — COMMENTED** [review]
After running a quick script, the declarations that I highlighted stood out.  Everything else appears in place.  Maybe.

File: Mathlib/Topology/ContinuousOn.lean (hunk 1 above)
Comment (j-loreaux) [reply Mathlib/Topology/ContinuousOn.lean:947]: This one was in the file before, but the arguments may have been in a slightly different order.

File: Mathlib/Topology/Defs/Filter.lean (hunk 2 above)
Comment (j-loreaux) [reply Mathlib/Topology/Defs/Filter.lean:163]: That's right, it didn't. I didn't mean to include it here, but it won't hurt anything. I'm going to tag a bunch of `ContinuousWithinAt` lemmas in an upcoming PR.

File: Mathlib/Tactic/FunProp/Continuous.lean (hunk 3 above)
Comment (j-loreaux) [reply Mathlib/Tactic/FunProp/Continuous.lean:215]: Yes, it was accidentally tagged twice.

File: Mathlib/Tactic/FunProp/Continuous.lean (hunk 5 above)
Comment (j-loreaux) [reply Mathlib/Tactic/FunProp/Continuous.lean:209]: Yes, it was accidentally tagged twice.

File: Mathlib/Tactic/FunProp/Continuous.lean (hunk 4 above)
Comment (j-loreaux) [reply Mathlib/Tactic/FunProp/Continuous.lean:221]: Yes, it was accidentally tagged twice.

**Comment by adomani** [comment]
@j-loreaux thanks for the responses!

**Comment by urkud** [comment]
I'm sorry I moved part of `Topology/Order/Basic` to a new file today. Please merge `master` and put it back on the queue.
bors r-
bors d+

**Comment by urkud** [comment]
I'm going to merge `master` now.

**Comment by urkud** [comment]
bors merge

**Comment by urkud** [comment]
bors merge

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**